includes/Interfaces/Renderer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Renderer Interface.
+ *
+ * Stories renderers should conform to this interface,
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Web_Stories\Interfaces;
+
+// Prevent direct access.
+defined( 'ABSPATH' ) || exit;
```

Comment:
This is not something we currently do in the code base, FWIW. Don't think we need to start doing that.